components/data.js

```diff
@@ -0,0 +1,380 @@
+const arrayWithKeys = [
```

Comment:
not ok naming concept - `array*` 
if we use key`S` - it means lots of smth - array by default...